assignment/api/src/java/org/sakaiproject/assignment/api/model/AssignmentSubmission.java

```diff
@@ -148,7 +148,7 @@
     private Boolean hiddenDueDate = Boolean.FALSE;
 
     @Column(name = "USER_SUBMISSION")
-    private Boolean userSubmission = Boolean.FALSE;
+    private Boolean userSubmission = Boolean.TRUE;
```

Comment:
This is should only be set to tru if the submission was submitted by the user.

Reply:
That's the case, though. The way it was developed originally it's only changed to false when it wasn't submitted by the user. So I thought it was fine and put it like it worked back on 11.x.

Reply:
A submission can be created by an instructor or by a student. This should only be set to true when created by the student which means it doesn't make sense that it is true by default.

Reply:
I know, what I meant was in 11.x it was the other way around. It was true by default, and only on those cases where the submission is not created by a student, it was set to false. So it was easier for us to just copy it from there, but if you prefer to change the behavior we can do it like that. Should we?

Reply:
Not sure if I made myself clear, so I'll try it again. There are >10 places where a submission is created/updated, but only on two of them it's needed to set that field to false (and it is set right now). So we thought it was better to keep it like that, but we can obviously change the behavior if you want us to.

Reply:
Yes I think it makes sense to change it since when an instructor creates an assignment they can click "view submissions/grade" and it creates a submission for every student by default. This mass creation of submissions is much more likely to be the case than a student creating the submission first.

Reply:
I am not a fan of this idea to create a submission for every student. I understand a submission is needed in order to send grade but i feel that could've been handled differently!